openlibrary/plugins/upstream/adapter.py

```diff
@@ -105,7 +105,9 @@ def process_error(self):
                 d['key'] = unconvert_key(d['key'])
             self.output = simplejson.dumps(d)
 
-def convert_key(key, mapping=convertions):
+
+def convert_key(key, mapping=None):
+    mapping = mapping or convertions
```

Comment:
Please move this line ___below___ the docstring for this function or the [doctests](https://docs.python.org/3/library/doctest.html) will be disabled.  We need all the tests we can get.  ;-)

Reply:
I need to move only line 110 or both 110 and 128

Reply:
128 also please.

Those `>>>` comments actually get run by pytest and the output is tested against the text on the following line.